openedx/core/djangoapps/ccxcon/api.py

```diff
@@ -16,9 +16,9 @@
 )
 
 from lms.djangoapps.courseware.courses import get_course_by_id
-from lms.djangoapps.instructor.access import list_with_level
 from openedx.core.djangoapps.models.course_details import CourseDetails
 from student.models import anonymous_id_for_user
+from student.roles import CourseInstructorRole
```

Comment:
If the tests pass, they pass, and this is waaaay outside the scope of this ticket, but I did notice that there are several places where openedex code references LMS code with the full path.